sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/PropertiesAccessCorrectFieldBase.cs

```diff
@@ -145,34 +171,23 @@ protected IList<PropertyData> CollectPropertyData(IEnumerable<IPropertySymbol> p
             return allPropertyData;
         }
 
-        /**
-         * Assignments can be done either
-         * - directly via an assignment
-         * - indirectly, when passed as 'out' or 'ref' parameter
-         */
-        protected abstract IEnumerable<FieldData> FindFieldAssignments(IPropertySymbol property, Compilation compilation);
-        protected abstract IEnumerable<FieldData> FindFieldReads(IPropertySymbol property, Compilation compilation);
-        protected abstract bool ImplementsExplicitGetterOrSetter(IPropertySymbol property);
-        protected abstract bool ShouldIgnoreAccessor(IMethodSymbol accessorMethod);
-
-
         protected struct PropertyData
```

Comment:
not really related to this commit but I noticed that we could make this: "private readonly struct" and it would be nice.